SimPEG/EM/Static/Utils/StaticUtils.py

```diff
@@ -207,7 +207,7 @@ def xy_2_r(x1, x2, y1, y2):
     dl_x = (endl[1, 0] - endl[0, 0]) / dl_len
     dl_y = (endl[1, 1] - endl[0, 1]) / dl_len
 
-    nstn = np.floor(dl_len / a)
+    nstn = int(np.floor(dl_len / a))
```

Comment:
@sgkang: I cast this to an `int` as as numpy ints can no longer be used to index arrays or initialize `np.ones()` or `np.zeros()`